README.md

```diff
@@ -1,85 +1,232 @@
-=======================================
-Djangoflow Chat
-=======================================
+# Djangoflow Chat
 
 Opinionated Django Chat
 
+## Requirements
 
+> **_NOTE:_**  Generated content, as with human generated content, may contain errors or oversights. Please review before merging!
 
+Before integrating `django-df-chat` into your Django project, ensure that the following prerequisites are met:
 
+1. [Python](https://www.python.org/): This project requires Python 3.8 or newer.
+2. [Django](https://www.djangoproject.com/): Django version 3.2 or newer is required.
+3. [DjangoFlow](https://github.com/djangoflow): Django Flow, our base project with integrated CI/CD, is a recommended starting point for incorporating `django-df-chat`.
+4. [Django REST Framework](https://www.django-rest-framework.org/): This project makes use of Django REST Framework for building APIs.
+5. [Django Channels](https://channels.readthedocs.io/en/latest/): This is needed for handling WebSockets and facilitating real-time communication.
+6. [Django Channels REST Framework](https://github.com/hishnash/djangochannelsrestframework): A library for building WebSocket API.
+7. [DRF Spectacular](https://drf-spectacular.readthedocs.io/en/latest/): A Django REST framework schema generation tool.
+8. [Django Hashid Field](https://github.com/nshafer/django-hashid-field): A reusable Django field that hides the fact that you're using AutoField or IntegerField primary keys in your objects.
+9. [DRF Nested Routers](https://github.com/alanjds/drf-nested-routers): This is used for managing CRUD operations on nested routes.
+10. [Django REST Framework SimpleJWT](https://django-rest-framework-simplejwt.readthedocs.io/en/latest/): This is used for authentication using JSON Web Tokens.
+11. [PyJWT](https://pyjwt.readthedocs.io/en/latest/): A Python library which allows you to encode and decode JSON Web Tokens (JWT).
+12. [Django Model Utils](https://django-model-utils.readthedocs.io/en/latest/): This project uses Django Model Utils for model mixins and utilities.
+13. [Django DF Notifications](https://github.com/dfurtado/django-notifications): This is used for managing user notifications in Django.
```

Comment:
Wrong link to DF notifications lib https://github.com/djangoflow/django-df-notifications